recipes/opentracing-cpp/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,14 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+find_package(OpenTracing REQUIRED CONFIG)
+
+add_executable(${PROJECT_NAME} test_package.cpp)
+if(TARGET OpenTracing::opentracing-static)
```

Should fix the compilation error

```suggestion
add_executable(${PROJECT_NAME} test_package.cpp)
set_property(TARGET ${PROJECT_NAME} PROPERTY CXX_STANDARD 11)
if(TARGET OpenTracing::opentracing-static)
```